config.json

```diff
@@ -69,6 +69,16 @@
       ]
     },
     {
+      "uuid": "6eba4eac-6665-4ad6-ac21-3651a11ab4b4",
+      "slug": "isogram",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 1,
+      "topics": [
+        "Data.Char"
```

Comment:
in contrast to Maybe, I don't think this:

1. affects where we would choose to place the exercise
2. is necessarliy going to be in everyone's solutions

so I would remove it

Reply:
right